web/src/typing_data.ts

```diff
@@ -3,11 +3,13 @@ import * as util from "./util";
 
 // See docs/subsystems/typing-indicators.md for details on typing indicators.
 
-const typist_dct = new Map<string, number[]>();
+const pm_typists_dict = new Map<string, number[]>();
+const stream_typists_dict = new Map<string, number[]>();
```

Comment:
Should we rename `stream_typists_dict` to `stream_topic_typists_dict` or more simply `topic_typists_dict`?

Reason: 

Currently our dict is `stream_typists_dict` and we use the term `get_topic_key`.

For PMs, it looks better: `pm_typists_dict` and `get_pms_key`

Reply:
`topic_typists_dict` seems clear enough to me, but I wouldn't object to `stream_topic_` or `stream_conversation_typists_dict`.